html/js/vr-party-participant.js

```diff
@@ -165,7 +175,20 @@ function viewersApplyState() {
     }
 
     if (_model_state.isolate_id !== undefined) {
-        viewersApply('isolate', _model_state.isolate_id);
+        var ids = _model_state.isolate_id;
+        if ((LMV_VIEWER_VERSION === '1.2.13' || LMV_VIEWER_VERSION === '1.2.14') &&
+            ids.length > 0 && typeof ids[0] === 'number') {
+                // getNodesByIds can throw an exception when the model isn't sufficiently loaded
+                // Catch it and try to apply the viewer state again in a second
+                 try {
+                    ids = _viewerLeft.model.getNodesByIds(ids);
+                }
+                catch (ex) {
+                    setTimeout(function() { viewersApplyState(); }, 1000);
+                    return;

```

Comment:
I stumbled across the return. Because in case "isolate" fails then slicing is not applied:
https://github.com/KeanW/vr-party/blob/f5621771dbbae5489a407efe4b9ec657da398e04/html/js/vr-party-participant.js#L195-L198

I wonder if we should set `ids` to undefined in the catch case and then for an undefined `ids` here:
https://github.com/KeanW/vr-party/blob/f5621771dbbae5489a407efe4b9ec657da398e04/html/js/vr-party-participant.js#L191-L192